app/models/user.rb

```diff
@@ -4,8 +4,10 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  validates :first_name, presence: true, length: { maximum: 30 }
-  validates :last_name, presence: true, length: { maximum: 30 }
+  AGE_RANGE = %w( 18-29 30-29 40-49 50-59 60-69 70+ ).freeze
```

Comment:
Please change to 30-39.

Reply:
Thanks for the catch! Please review updated commit.